### PR TITLE
[1/n] Global parameters renderer

### PR DESCRIPTION
**Pull request by saqadri**
[1/n] Global parameters renderer

Use ParametersRenderer to modify global parameters.

Currently still updating the entire parameters object. Next up, will fix the ParametersRenderer component to update each parameter separately.

### CODE REVIEW

**Comment by saqadri** [comment]
Test Plan:
```
pip install -e python/
alias aiconfig="python3 -m 'aiconfig.scripts.aiconfig_cli'"
```

```
aiconfig edit --aiconfig-path=/Users/saqadri/lm/aiconfig/python/src/aiconfig/editor/travel.aiconfig.json --server-port=8080 --server-mode=debug_servers
```

https://github.com/lastmile-ai/aiconfig/assets/25641935/8f238b55-c34f-409a-ba85-7f72acc61b3c

File: python/src/aiconfig/editor/client/src/components/GlobalParametersContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    },
+    paddingBottom: 50,
+  },
+}));
+
+export default memo(function GlobalParametersContainer({
+  initialValue,
+  onUpdateParameters,
+}: Props) {
+  const [isParametersDrawerOpen, setIsParametersDrawerOpen] = useState(false);
```
Comment (rholinshead) [inline python/src/aiconfig/editor/client/src/components/GlobalParametersContainer.tsx:24]: What's the LOE to make this component UI into the card with bottom show/hide like in the figma -- https://www.figma.com/file/7YVa6KpEDgptj8DiAc5BaU/AiConfig-Editor?type=design&node-id=328-5037&mode=design&t=h3xAGY47py1mg4Fk-0 ?

**Review by rholinshead — APPROVED** [review]
Functionality LGTM and we can iterate on the UI itself to better match the designs

File: python/src/aiconfig/editor/client/src/components/GlobalParametersContainer.tsx (hunk 1 above)
Comment (saqadri) [reply python/src/aiconfig/editor/client/src/components/GlobalParametersContainer.tsx:24]: Probably not very high. I can do that as a follow-up next week

**Pull request merged**